lib/components/admin/field-trip-details.js

```diff
@@ -8,6 +8,7 @@ import { format, formatDistanceToNow, parse } from 'date-fns'
 import {
   getDateFormat,
   // yyyy-MM-dd, which is same format required for date input controls.
+  OTP_API_DATE_FORMAT,
   OTP_API_DATE_FORMAT_DATE_FNS
```

Comment:
Out of curiosity, does `OTP_API_DATE_FORMAT_DATE_FNS` still need to be imported if it's been removed below?

Reply:
Good spot @amy-corson-ibigroup, thanks, that constant doesn't exist anymore so it can be removed.